src/main/java/org/folio/inventory/domain/items/LastCheckIn.java

```diff
@@ -0,0 +1,128 @@
+
+package org.folio.inventory.domain.items;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonObject;
+import java.util.Date;
+import org.joda.time.format.ISODateTimeFormat;
+
+/**
+ * Information about when an item was last scanned in the Inventory app.
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder( {
+    "dateTime",
+    "servicePointId",
+    "staffMemberId"
+})
+public class LastCheckIn {
```

Comment:
Is this generated code?

If not, why does this class need to mutable? Why does it need JSON property attributes? Is all of that added only to avoid implementing the `toJson` method manually?

Reply:
It is generated by RMB and copied here.

Reply:
I would prefer we didn't copy and paste generated code from other modules without changing it.

Please can this be amended to better fit within the style of mod-inventory / mod-circulation.

Personally, I would do that by making it immutable (with constructor parameters) and manually implementing the `toJson` method.

Reply:
Done.

Reply:
Thank you for changing the implementation of this class. I think that means that the comments and the annotations can also be removed?

Reply:
Sure